src/modules/75-cf/hooks/useEnvironmentSelectV2.tsx

```diff
@@ -107,18 +113,27 @@ export const useEnvironmentSelectV2 = (params: UseEnvironmentSelectV2Params) =>
           usePortal={false}
           popoverClassName={!showCreateButton ? css.hideCreateButton : ''}
           value={selectedEnvironment}
-          items={selectOptions}
+          items={
+            ALL_ENVIRONMENTS_ENABLED && allowAllOption && selectOptions.length > 1
```

Comment:
I think the guidance now is that we allow this view even when there is only a single env (makes no sense to me personally)
```suggestion
            ALL_ENVIRONMENTS_ENABLED && allowAllOption
```

Reply:
I'll get confirmation on that and update if that's what they want

Reply:
Honestly, I think showing it when there's only a single env is pointless, so I hope they say to leave as-is 👍🏻